### PR TITLE
Small simplifications and speed-ups

### PR DESCRIPTION
**Pull request by TimFelixBeyer**
Just some small tweaks in the spirit of #1631 and #1683 that don't change the functionality.

### CODE REVIEW

**Comment by mscuthbert** [comment]
closing and reopening to update lint.

File: music21/scale/__init__.py [hunk 1]
```diff
@@ -1321,10 +1321,7 @@ def isConcrete(self):
         To be concrete, a Scale must have a
         defined tonic. An abstract Scale is not Concrete
         '''
-        if self.tonic is None:
-            return False
-        else:
-            return True
+        return self.tonic is not None
```
Comment (mscuthbert) [inline music21/scale/__init__.py:1324]: please stop changing these.  I've asked a couple of times already.

File: music21/scale/__init__.py [hunk 2]
```diff
@@ -1358,15 +1355,14 @@ def __eq__(self, other):
...
-            if (isinstance(other, self.__class__)
-                    and isinstance(self, other.__class__)
-                    and self._abstract == other._abstract
-                    and self.boundRange == other.boundRange
-                    and self.tonic == other.tonic):
-                return True
-            else:
-                return False
+
+        if (isinstance(other, self.__class__)
```
Comment (mscuthbert) [inline music21/scale/__init__.py:1359]: as above.  Please stop changing code style that does not increase speed, just to match your code style.

**Review by mscuthbert — CHANGES_REQUESTED** [review]
Hi Tim -- really great 90%, but I've asked a couple of times to stop changing code style just because it could be written in another way, and I'm now going to hold off on the PR until those lines are changed back.  They don't save the processor any significant time, but they eat up my time in reviewing.

**Comment by mscuthbert** [comment]
THANKS!  Huge appreciation.  Merging!

**Review by mscuthbert — APPROVED** [review]

**Comment by TimFelixBeyer** [comment]
> Hi Tim -- really great 90%, but I've asked a couple of times to stop changing code style just because it could be written in another way, and I'm now going to hold off on the PR until those lines are changed back. They don't save the processor any significant time, but they eat up my time in reviewing.

I’m not doing this to annoy you, I usually just work with a very heavily modified version of m21 and sometimes some of my other changes slip in. I will be more careful when making PRs to music21 main in the future to make sure only the most important changes are included.

**Pull request merged**